mutt_commands.h

```diff
@@ -72,6 +73,11 @@ enum MuttSetCommand
 /* parameter to parse_mailboxes */
 #define MUTT_NAMED   (1 << 0)
 
-extern const struct Command Commands[];
+/* command registry functions */
+#define COMMANDS_REGISTER(cmds) commands_register(cmds, sizeof(cmds) / sizeof(cmds[0]))
```

Comment:
We have `mutt_array_size1 in `mutt/memory.h`

Reply:
Using it now, thanks for the hint.